### DISCUSSION TITLE
Step 3: "Open completions menu" option does not show in CoPilot suggestions

### DISCUSSION THREAD

**Discussion by SimonDarksideJ** [skills/.github#288] [Code with Copilot]
In step 3, while I could navigate through the suggestions using the arrows, the menu option ("Open completions menu") shown in the image and instructions did not show.

![image](https://github.com/skills/.github/assets/1793042/c29e04f5-099c-4c2d-b3e8-a07ddd284ff8)

**Comment by Peace-Aielumoh** [comment]
Hi, Have you tried pressing Ctrl + Enter

**Reply by aditya-524** [reply]
Thanks for this.

**Comment by SimonDarksideJ** [comment]
Yes, Ctrl+Enter does work (thankfully @Peace-Aielumoh  ) so it is possible, however, in the initial stages this is not proposed, with the reader only having the screenshot as a reference, which does not work.

**Comment by racheltienng** [comment]
Hello,

I had the same issue. I had to zoom in the screenshot to get to the Ctrl+Enter (I use Mac), and then it opened the GitHub Copilot 2 tab. Then I could click on the accept solution button.

I agree with Simon that the wording for this step is very vague and hard to understand.

**Comment by ghost** [comment]
hi I have been able to commit my skills.js file to my GitHub but step 3 did not pop up

**Reply by ghost** [reply]
found it!

**Reply by sergiosolorzano** [reply]
Ctrl+Enter showed me the proposed solution but Open Completions Menu doesn't show so can't select anything but accept that solution shown to move forward, can't see any other proposed code

**Comment by mateuszstruzik** [comment]
maybe will  be good idea to add this hint ctrl + enter in description
